docs/guides/configuration-guide/loadbalancer.md

```diff
@@ -83,16 +122,44 @@ is known everywhere and the self-signed certificates are accepted as valid.
    ```
 
 3. Use in OpenStack
+   * Add the custom CA to the configuration repository in the directory `environments/kolla/certificates/ca` with the same
+     name like in step 1
+   * Configure the custom CA to be copied to the OpenStack containers
+     ```yaml title="environments/manager/configuration.yml"
+     kolla_copy_ca_into_containers: "yes"
+     openstack_cacert: /etc/ssl/certs/ca-certificates.crt
+     ```
+
+4. Import the ca certificate to all nodes so that the custom CA is known everywhere and the self-signed certificates are accepted as valid.
+   ```
+   osism apply certificates
+   ```
+
+5. Execute all steps in the general documentation above
```

Comment:
This is also unspecific, there is nothing labeled in the above docs as being "general", which steps precisely?